network-api/networkapi/wagtailpages/models.py

```diff
@@ -502,12 +502,25 @@ class BanneredCampaignPage(PrimaryPage):
         blank=True,
         null=True,
         on_delete=models.SET_NULL,
-        help_text='Choose existing or create new sign-up form'
+        help_text='Choose an existing, or create a new, pettition form'
     )
 
-    content_panels = PrimaryPage.content_panels + [
-        SnippetChooserPanel('cta')
-    ]
+    signup = models.ForeignKey(
+        'Signup',
+        related_name='bcpage',
+        blank=True,
+        null=True,
+        on_delete=models.SET_NULL,
+        help_text='Choose an existing, or create a new, sign-up form'
+    )
+
+    panel_count = len(PrimaryPage.content_panels)
```

Comment:
"put things in between other panels" - this adds the signup option between the existing cta option, and the body field.